core/nodepool/config/templates/stack-template.json

```diff
@@ -361,6 +368,7 @@
                   "Action": [
                     "autoscaling:DescribeAutoScalingGroups",
```

Comment:
@redbaron 
> On larger clusters you'd have large controller nodes too :)

Yea, that's certainly true!
Do you think we can just recommend users to make controller large enough, instead of complication cluster.yaml like this?